checks/apps/pytorch/pytorch_nvidia.py

```diff
@@ -1,36 +1,68 @@
 import sys
 import pathlib
 import reframe as rfm
+import reframe.utility.sanity as sn
+import requests
+import re
+from packaging.version import Version
 from pytorch_test_base import PyTorchTestBase
-
 sys.path.append(str(pathlib.Path(__file__).parent.parent.parent / 'mixins'))
 from container_engine import ContainerEngineMixin  # noqa: E402
 
 
+
+def latest_nvidia_pytorch_image_tags():
```

Comment:
This is really useful and we might need it also for other tests. I suggest you take it out and add it in a utility file outside of the test. Maybe we could also make it more generic and let it accept some image/tag?

Reply:
Sure, sounds like a great suggestion.